xrdp-0.10.0.tar.gz does not include 'docs/man/xrdp-mkfv1.8.in'.

### xrdp version

0.10.0

### Detailed xrdp version, build options

~~~
./configure --enable-jpeg --enable-tjpeg --enable-pixman --enable-opus --enable-mp3lame --enable-rdpsndaudin --enable-pamuserpas --with-imlib2=yes --enable-vsock --with-freetype2=yes
~~~


### Operating system & version

Ubuntu 22.04

### Installation method

other

### Which backend do you use?

_No response_

### What desktop environment do you use?

_No response_

### Environment xrdp running on

_No response_

### What's your client?

_No response_

### Area(s) with issue?

Compile error

### Steps to reproduce

~~~
wget https://github.com/neutrinolabs/xrdp/releases/download/v0.10.0/xrdp-0.10.0.tar.gz
tar xzvf xrdp-0.10.0.tar.gz
cd xrdp-0.10.0/
./bootstrap
./configure --enable-jpeg --enable-tjpeg --enable-pixman --enable-opus --enable-mp3lame --enable-rdpsndaudin --enable-pamuserpas --with-imlib2=yes --enable-vsock --with-freetype2=yes
make
~~~

### ✔️ Expected Behavior

make successfully.

### ❌ Actual Behavior

~~~
make[3]: *** No rule to make target 'xrdp-mkfv1.8.in', needed by 'xrdp-mkfv1.8'.  Stop.
~~~

### Anything else?

xrdp-0.10.0.tar.gz does not contain 'docs/man/xrdp-mkfv1.8.in'.
v0.10.0.tar.gz contains "docs/man/xrdp-mkfv1.8.in".

xrdp-0.10.0.tar.gz is missing the required files.

--- a/docs/man/Makefile.am
+++ b/docs/man/Makefile.am
@@ -1,9 +1,3 @@
-if USE_FREETYPE2
-  MKFV1_MAN = xrdp-mkfv1.8
-else
-  MKFV1_MAN =
-endif
-
 man_MANS = \
   xrdp-dis.1 \
   sesman.ini.5 \
@@ -16,10 +10,13 @@ man_MANS = \
   xrdp-sesadmin.8 \
   xrdp-sesman.8 \
   xrdp-sesrun.8 \
-  xrdp-dumpfv1.8 \
-  $(MKFV1_MAN)
+  xrdp-dumpfv1.8
 
-EXTRA_DIST = $(man_MANS:=.in)
+EXTRA_DIST = xrdp-mkfv1.8.in $(man_MANS:=.in)
+
+if USE_FREETYPE2
+  man_MANS += xrdp-mkfv1.8
+endif
 
 SUBST_VARS = sed \
    -e 's|@PACKAGE_VERSION[@]|$(PACKAGE_VERSION)|g' \

